pyne/source_sampling.pyx

```diff
@@ -187,6 +189,31 @@ cdef class AliasTable:
     pass
 
 
+def convert_e_bounds(e_bounds):
+    """convert_e_bounds(rands)
+    Convert python np.ndarray into vector< double> 
+    
+    Parameters
+    ----------
+    e_bounds : np.ndarray
+    
+    Returns
+    -------
+    res1 : std::vector< double >
```

Comment:
This does not return a std::vector because it is a Python function.  Cython is automatically converting the vector into a Python list on return.  You need to make to make this a cdef function and add a return type to have this return a std::vector.  See http://cython.readthedocs.io/en/latest/src/userguide/language_basics.html#python-objects-as-parameters-and-return-values

Reply:
I found that we don't need this function anymore as we implemented the properties for all the x, y, z, e, w and c.

Reply:
Ok, if we don't need this function, then  please delete it :)